server/matchmaker/search.py

```diff
@@ -16,6 +17,28 @@
 OnMatchedCallback = Callable[["Search", "Search"], Any]
 
 
+class Game:
+    """
+    Represents a potential game
+    """
+    def __init__(
+        self,
+        match: Match,
+        match_quality: float
+    ):
+        self.match = match
+        self.quality = match_quality
+
+
+def get_average_rating(searches):
+    ratings = []
+    for s in searches:
+        for mean, dev in s.raw_ratings:
+            rating = mean - 3 * dev
+            ratings.append(rating)
+    return statistics.mean(ratings)
```

Comment:
This could be a one-liner.
```suggestion
    return statistics.mean(mean - 3 * dev for s in searches for mean, dev in s.raw_ratings)
```